traitcuration/traits/datasources/dummy.py

```diff
@@ -0,0 +1,148 @@
+"""
+This module holds dummy data to import during development
+"""
+from ..models import OntologyTerm, MappingSuggestion, Trait, Mapping, User, Review
+
+
+def import_dummy_data():
+    Review.objects.all().delete()
+    MappingSuggestion.objects.all().delete()
+    Mapping.objects.all().delete()
+    Trait.objects.all().delete()
+    OntologyTerm.objects.all().delete()
+    User.objects.all().delete()
+    # ONTOLOGY TERMS
+    term_current1 = OntologyTerm(label='Diabetes mellitus', curie='EFO:0000400',
```

Comment:
I suggest renaming the dummy traits so that it is obvious they do not really come from ClinVar and have been pre-populated with data. This can be done by e.g. naming them '/ Diabetes mellitus /' or in some other way. This will not prevent them from being updated by ZOOMA, but will clearly mark and distinguish them.